esp/src/eclwatch/ESPLogicalFile.js

```diff
@@ -84,10 +84,15 @@ define([
             }
         },
         preProcessRow: function (item, request, query, options) {
+            var convertNull;
+            if (item.IntSize === null) {
+                convertNull = 0;
+            }
             lang.mixin(item, {
                 __hpcc_id: createID(item.NodeGroup, item.Name),
                 __hpcc_isDir: false,
-                __hpcc_displayName: item.Name
+                __hpcc_displayName: item.Name,
+                Size: convertNull //superfiles causing issues sorting by returning null
```

Comment:
The `IntSize: item.IntSize ? item.IntSize : 0` is ok but there are sometimes the value is null and it is appropriate to display null to the user but the store has to think of it as 0 to properly sort by this value.